### PR TITLE
TWIRF - More Tickets!

### PR DESCRIPTION
**Pull request by Hoverbear**
@RustFestEU/zurich-2017 Please review and add what you need

### CODE REVIEW

**Comment by gandro** [comment]
LGTM! Thanks! 

Please ping me once the text is finalized, so I can copy-paste it into the the newsletter for tomorrow morning :)

**Comment by dns2utf8** [comment]
I added a speaker FAQ. Would you like to add something?

File: _posts/2017-08-22-this-week-in-rustfest-8-speakers.md [hunk 1]
```diff
@@ -102,3 +102,15 @@ Yeah, bet you thought you'd only get 0 or 1 poems didn't you? Well you get **two
 Finally, we've confirmed one of our workshops! **RustBridge** hosted by [**Ashley Williams**](https://github.com/ashleygwilliams) will be a part of our Sunday workshop day in Zürich. This workshop will be catered towards people new to the Rust community. If this is your first blush with Rust consider attending this workshop to help you hit the ground running.
 
 We'll be announcing further speakers (and putting up a nice schedule) soon! Keep your eyes peeled.
+
+# FAQ for speakers
+
+As we are looking forward to the talks some technical questions arrise.
```
Comment (Hoverbear) [inline _posts/2017-08-22-this-week-in-rustfest-8-speakers.md]: `arrise` -> `arise`

File: _posts/2017-08-22-this-week-in-rustfest-8-speakers.md [hunk 2]
```diff
@@ -102,3 +102,15 @@ Yeah, bet you thought you'd only get 0 or 1 poems didn't you? Well you get **two
...
+Here is our current best of speaker FAQ:
```
Comment (Hoverbear) [inline _posts/2017-08-22-this-week-in-rustfest-8-speakers.md]: "Here are some answers to questions you might have:"

(Using 'best' in this case makes it sound like you aren't happy with it, which you might not be, but that's not a great way to communicate confidence/organization/authority)

File: _posts/2017-08-22-this-week-in-rustfest-8-speakers.md [hunk 3]
```diff
@@ -102,3 +102,15 @@ Yeah, bet you thought you'd only get 0 or 1 poems didn't you? Well you get **two
...
+
+* The beamer will have a
+    * 16:9 ration with the resolution 1920x1080, please fill the screen
+    * HDMI and VGA connection
+* There will be WiFi all over the campus
+* You can borrow a presenter from us, nothing fancy
```
Comment (Hoverbear) [inline _posts/2017-08-22-this-week-in-rustfest-8-speakers.md]: What do you mean by presenter?

File: _posts/2017-08-22-this-week-in-rustfest-8-speakers.md [hunk 4]
```diff
@@ -102,3 +102,15 @@ Yeah, bet you thought you'd only get 0 or 1 poems didn't you? Well you get **two
...
+* The time must be adhered. We have a packed schedule
```
Comment (Hoverbear) [inline _posts/2017-08-22-this-week-in-rustfest-8-speakers.md]: "* If you go over time we may cut you short, so please be timely."

**Comment by Hoverbear** [comment]
@dns2utf8 Please add yourself to authors.

File: _posts/2017-08-22-this-week-in-rustfest-8-speakers.md (hunk 3 above)
Comment (dns2utf8) [reply _posts/2017-08-22-this-week-in-rustfest-8-speakers.md]: a presenter is a remote keyboard with two keys. Forward and backward

**Review by Hoverbear — COMMENTED** [review]
LGTM

**Comment by gandro** [comment]
I scheduled this as a newsletter for tomorrow morning, 11:00am CEST.

**Pull request merged**